test/parallel/test-querystring-maxKeys-non-finite.js

```diff
@@ -0,0 +1,49 @@
+'use strict';
+// This test was originally written to test a regression
+// that was introduced by
+// https://github.com/nodejs/node/pull/2288#issuecomment-179543894
+require('../common');
+
+const assert = require('assert');
+const parse = require('querystring').parse;
+
+// taken from express-js/body-parser
```

Comment:
put in a full github url including hash for the exact version you're pulling from